istioctl/cmd/proxyconfig.go

```diff
@@ -299,7 +300,18 @@ func clusterConfigCmd() *cobra.Command {
 			var configWriter *configdump.ConfigWriter
 			var err error
 			if len(args) == 1 {
-				podName, ns := handlers.InferPodInfo(args[0], handlers.HandleNamespace(namespace, defaultNamespace))
+				var client kube.ExtendedClient
+				client, err = kubeClient(kubeconfig, configContext)
```

Comment:
Even though each subcommand run independently of each other, they all create an instance of `kubeClient`.
Is there a way we can create `kubeClient` once and share it across subcommands? May be using `PersistentPreRun` ?

Reply:
Thanks, I decided to put this in `Args` as it is part of argument parsing.